pyamg/classical/tests/test_classical.py

```diff
@@ -12,7 +12,8 @@
 
 from pyamg.classical import split
 from pyamg.classical.classical import ruge_stuben_solver
-from pyamg.classical.interpolate import direct_interpolation
+from pyamg.classical.interpolate import direct_interpolation, \
+       distance_two_interpolation, standard_interpolation
 
```

Comment:
@bensworth @lukeolson We need to add tests for extended and extended+i.  I'm not sure how to do this.  A few ideas.

1. The paper https://onlinelibrary.wiley.com/doi/abs/10.1002/nla.559 
lists a reference stencil for the 1D Poisson operator and
-  ext+i  in equation (16), `w_1,0 = 2/3, w_1,3 = 1/3`)
- ext at the very top of section 4.5, `w_1,0 = 0.5, w_1,3 = 0.5`

2. Compare to specific interp stencils produced by hypre
3. Compare to macro statistics (like op complexity and convergence) produced by hypre